doc/nrf/releases/release-notes-latest.rst

```diff
@@ -113,16 +171,43 @@ The following list summarizes the most important changes inherited from upstream
   * Updated the Controller to only use control procedures supported by the peer.
   * Added support for the nRF52820 SoC.
   * Removed the legacy Controller.
+  * Implemented a function to remove auxiliary advertising sets (``ll_adv_aux_set_remove()``).
+  * Implemented a function to remove all primary channels and auxiliary channels of an advertising set (``ll_adv_aux_set_clear()``).
+  * Fixed overflow that could happen when using uninitialized PDU.
+  * Removed redundant :option:`CONFIG_BT_LL_SW_SPLIT` conditional.
+  * Enforced support for Read RSSI command if the Connection State is supported.
```

Comment:
Is "enforced support" correct here? Or should it be "added support", or "Enforced that the Read RSSI command is used if..."/"Added code to enforce that ..."?

Reply:
`Enforced that the Read RSSI command is supported if the Connection State is supported.`
This is BLE standard requirement.